presto-spi/src/main/java/com/facebook/presto/spi/block/Block.java

```diff
@@ -295,4 +295,12 @@ default Block getLoadedBlock()
     {
         return this;
     }
+
+//    int[] getSizesInBytes();
+//
+//    int[] getSizesInBytes(int[] offsets, int offset, int length);
+
+    void accumulateSizesInBytes(int[] outputSizesInBytes);
+    void accumulateSizesInBytes(int[] upperLevelOffsets, int[] outputSizesInBytes);
```

Comment:
I'd rename these methods to match `getRegionSizeInBytes` and `getPositionsSizeInBytes`.

```
void appendPositionSizesInBytes(int[] sizesInBytes);

void appendRegionSizesInBytes(int[] offsets, int[] sizesInBytes);
```